.github/workflows/legal-api-cd.yml

```diff
@@ -20,15 +15,15 @@ defaults:
 
 env:
   APP_NAME: "legal-api"
-  TAG_NAME: "dev"
+  TAG_NAME: "test"
 
 jobs:
   legal-api-cd-by-push:
     runs-on: ubuntu-20.04
 
-    if: github.event_name == 'push' && github.repository == 'bcgov/lear'
+    if: github.event_name == 'workflow_dispatch' && github.repository == 'bcgov/lear'
     environment:
-      name: "dev"
+      name: "${{ github.event.inputs.environment }}"
```

Comment:
Here I'm hoping this will build to Test environment (with the "test" tag).

I may have to disable the "legal-api-cd-by-dispatch" job below for Test and allow it only for Prod. Thoughts?